R/calc_water_year.R

```diff
@@ -8,6 +8,7 @@ calc_water_year <- function(date.vec){
   which.past.oct <- cal.mon >= 10
   
   # add one to the year if it is in October or after
+  # October is end of water year
```

Comment:
oh wait nevermind, that's what @limnoliver added :sweat_smile: phew not in a feature pkg.